src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb

```diff
@@ -784,45 +784,6 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
             End If
         End Function
 
-
-        ''' <summary>
-        ''' Check if there's WCF import/generation error/warning. If there's one, force to display the error list page
```

Comment:
Basically ShellUtil.vb is copied 4 times in VS. Twice in our tree (I'm looking at unifying), twice in the VS tree. Most of the helpers aren't used by most of the projects.